news/TEMPLATE

```diff
@@ -8,7 +8,7 @@
 
 ### Deprecations
 
-* <news item>
+* Deprecate `conda_build.exceptions.UnableToParseMissingJinja2`. (#5497)
```

Comment:
Oops, looks like this news file needs to be duplicated and renamed! (it's currently the template file)

Reply:
🤦🏻‍♂️